### ISSUE TITLE
oop php5 deprecated on 5.5+ versions

### ISSUE THREAD

**Issue by vatnoise** [RichardVasquez/kses#3]
Hi, it seems that some bits of the code were deprecated in 5.6 and 5.5 versions.

```
Deprecated: preg_replace(): The /e modifier is deprecated, use preg_replace_callback instead in /var/www/...path.../php5.class.kses.php on line 542

Deprecated: preg_replace(): The /e modifier is deprecated, use preg_replace_callback instead in /var/www/...path.../php5.class.kses.php on line 158
```

**Comment by RichardVasquez** [comment]
I'm not doing a lot of PHP these days.  If you have a PR for it, I'll test it and merge it in.

**Comment by LinusU** [comment]
Should be fixed by #6
